docs/rules/prefer-toHaveBeenCalledWith.md

```diff
@@ -0,0 +1,45 @@
+# Prefer toHaveBeenCalledWith
+
+This rule recommends using `toHaveBeenCalledWith` instead of `toHaveBeenCalled`.
+## Rule details
+
+This rule is turned off by default (is set to **0**).
+
+The following patterns are considered a warning:
+
+
+```js
+describe("", function() {
+  it("", function() {
+    f(1);
+    expect(f).toHaveBeenCalled();
+  });
+});
+```
+
+```js
+describe("", function() {
+  it("", function() {
+    expect(f).not.toHaveBeenCalled();
+  });
+});
+```
+
+The following patterns are not warnings:
```

Comment:
let's add one to the docs/tests with `toHaveBeenCalledWith()` in case no args are needed